lib/process-services-cloud/src/lib/form/form-cloud.module.ts

```diff
@@ -39,7 +38,6 @@ export const FORM_CLOUD_DIRECTIVES = [
     FileViewerWidgetComponent,
     FilePropertiesTableCloudComponent,
     FormCustomOutcomesComponent,
-    FormDefinitionSelectorCloudComponent,
```

Comment:
This is deprecated now so I believe this should be reverted until that component will be removed